relayer/chains/evm/client.go

```diff
@@ -9,55 +9,43 @@ import (
 	ethereum "github.com/ethereum/go-ethereum"
 	"github.com/ethereum/go-ethereum/accounts/abi/bind"
 	ethTypes "github.com/ethereum/go-ethereum/core/types"
-	"github.com/ethereum/go-ethereum/rpc"
 	bridgeContract "github.com/icon-project/centralized-relay/relayer/chains/evm/abi"
-	"github.com/icon-project/centralized-relay/relayer/chains/evm/types"
-	providerTypes "github.com/icon-project/centralized-relay/relayer/types"
-	"github.com/pkg/errors"
 	"go.uber.org/zap"
 
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/ethclient"
 )
 
 const (
-	RPCCallRetry           = 5
-	MaxTxFixtures          = 5
-	DefaultMinedTimeout    = time.Minute * 5
-	DefaultPollingInterval = time.Second * 5
-	MaximumWaitTry         = 60
+	DefaultPollingInterval = time.Second * 30
+	MaximumPollTry         = 15
 )
 
 func newClient(ctx context.Context, connectionContract, XcallContract common.Address, rpcUrl, websocketUrl string, l *zap.Logger) (IClient, error) {
-	clrpc, err := rpc.Dial(rpcUrl)
-	if err != nil {
-		return nil, err
-	}
-	cleth, err := ethclient.DialContext(ctx, websocketUrl)
+	eth, err := ethclient.DialContext(ctx, websocketUrl)
```

Comment:
the variable name `eth` is not suitable. Let's have proper naming that is meaningful.

Reply:
Which will be better?
ethWebSocketClient, ethClient or ethClientRpc?
Any of the options looks good, I just choose to have shorter and understandable, I guess that's enough.